### PR TITLE
Add initramfs and readonly rootfs

### PR DESCRIPTION
**Pull request by varder**

### CODE REVIEW

File: recipes-core/images/aos-image-minimal.bb [hunk 1]
```diff
@@ -31,3 +40,14 @@ IMAGE_INSTALL_append = " \
     openssh \
     tzdata \
 "
+
+python quotaon() {
+  newinit = None
+  with open(d.expand('${IMAGE_ROOTFS}/lib/systemd/system/quotaon.service'), 'r') as init:
+    newinit = init.read()
+    newinit = newinit.replace('ExecStart=/usr/sbin/quotaon -aug', 'ExecStartPre=/usr/sbin/quotacheck -avum \nExecStart=/usr/sbin/quotaon -avu')
```
Comment (oleksiimoisieiev) [inline recipes-core/images/aos-image-minimal.bb:48]: Please user systemd-quotacheck default service to perform quotacheck

**Review by oleksiimoisieiev — CHANGES_REQUESTED** [review]

File: recipes-core/images/aos-image-minimal.bb [hunk 2]
```diff
@@ -16,6 +16,15 @@ IMAGE_ROOTFS_SIZE ?= "16384"
 IMAGE_ROOTFS_EXTRA_SPACE = "1048576"
 IMAGE_ROOTFS_EXTRA_SPACE_append = "${@bb.utils.contains("DISTRO_FEATURES", "systemd", " + 4096", "" ,d)}"
 
+ROOTFS_POSTPROCESS_COMMAND +=  "read_only_rootfs_hook; "
+APPEND_append = '" ro"'
```
Comment (oleksiimoisieiev) [inline recipes-core/images/aos-image-minimal.bb:20]: IMAGE_FEATURES_append = " read-only-rootfs"

File: recipes-core/systemd/systemd_%.bbappend [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+do_install_append () {
+    sed -i -e "s/\/lib\/systemd\/systemd-quotacheck/\/usr\/sbin\/quotacheck -avum/" \
+         ${D}${systemd_unitdir}/system/systemd-quotacheck.service
+
+    sed -i -e "s/quotaon -aug/quotaon -avu/" \
```
Comment (oleksiimoisieiev) [inline recipes-core/systemd/systemd_%.bbappend:6]: I'd rather leave -aug. Preserving group is more important then verbosity.

File: recipes-extended/netconfig/netconfig.bb [hunk 4]
```diff
@@ -14,7 +14,7 @@ do_install_append() {
     install -m 0644 ${WORKDIR}/*.network ${D}${sysconfdir}/systemd/network
 }
 
-pkg_postinst_ontarget_${PN} () {
+pkg_postinst_${PN} () {
```
Comment (oleksiimoisieiev) [inline recipes-extended/netconfig/netconfig.bb:17]: Could you please share the background of this change? I suppose this wouldn't be working as expected.

**Review by oleksiimoisieiev — APPROVED** [review]

**Review by Eabramov84 — APPROVED** [review]

**Comment by al1img** [comment]
New PR will be submitted.

**Pull request closed without merging**